### PR TITLE
Fix some more golint warnings.

### PR DESCRIPTION
**Pull request by prashanthpai**
Signed-off-by: Prashanth Pai <ppai@redhat.com>

### CODE REVIEW

**Comment by prashanthpai** [comment]
@kshlm This PR still doesn't fix warnings in `transaction` package. There are some name conflicts in the way structs have been named (txnCtx and TxnCtx). It'll probably be best if you can take a look at those.

~~~
./transaction/context.go:35:15: exported func NewCtx returns unexported type *transaction.txnCtx, which can be annoying to use
./transaction/context.go:42:45: exported func NewCtxWithLogFields returns unexported type *transaction.txnCtx, which can be annoying to use
./transaction/context.go:50:1: comment on exported function NewCtxWithPrefix should be of the form "NewCtxWithPrefix ..."
./transaction/context.go:51:38: exported func NewCtxWithPrefix returns unexported type *transaction.txnCtx, which can be annoying to use
Found 4 lint suggestions; failing.
./transaction/context_mock.go:13:1: exported function NewMockCtx should have comment or be unexported
./transaction/context_mock.go:13:19: exported func NewMockCtx returns unexported type *transaction.mockTxnCtx, which can be annoying to use
Found 2 lint suggestions; failing.
~~~

File: commands/peers/peer-rpc-svc.go [hunk 1]
```diff
@@ -31,7 +33,7 @@ var (
 	etcdConfFile = etcdConfDir + "etcdenv.conf"
 )
 
-// ValidateAdd() will checks all validation for AddPeer at server side
+// ValidateAdd will checks all validation for AddPeer at server side
```
Comment (kshlm) [inline commands/peers/peer-rpc-svc.go]: The grammar here sucks. "...will checks.." !.
Could you please reword this and the stuff below, along the lines of `Validate* validates ....`.

**Comment by kshlm** [comment]
I had just begun doing this for my PR #160. I'll fix the warnings in the transaction package with that.

File: commands/peers/peer-rpc-svc.go (hunk 1 above)
Comment (prashanthpai) [inline commands/peers/peer-rpc-svc.go]: Done.

**Pull request merged**